meetings/2020/WASI-10-22.md

```diff
@@ -44,3 +44,161 @@ Installation is required, see the calendar invite.
     4. Other ideas?
 
 ## Meeting Notes
+
+### Attendees
+
+- Sam Clegg
+- Thomas Lively
+- Martin Duke
+- Syrus Akbary
+- Pat Hickey
+- Tanya Crenshaw
+- Alon Zakai
+- Andrew Brown
+- Mingqiu Sun
+- Alex Crichton
+- Dan Gohman
+- Steven Dabell
+- David Piepgrass
+- Johnnie Birch
+- Martin Duke
+- Yong
+- Sergey Rubanov
+- Arun Purushan
+
+### Find volunteers for note taking
+
+TL volunteers
+
+### Adoption of the agenda
+
+PH seconds
+
+### Notes
+
+#### Review of action items from prior meeting
+
+DG: Next time ??? is in the meeting we will check up on ???
+
+#### Vote to move wasi-nn to stage 2
+
+- Phases: https://github.com/WebAssembly/WASI/blob/master/docs/Process.md
+
+AB: We proposed wasi-nn a while back. Have a witx spec. I’ve been creating a POC
+to see if this works. When looking at docs, realized we are essentially in stage
+2. Moving to phase 2 would reflect reality. Asking for feedback on API.
+
+SC: English spec text requirement doesn’t apply?
+
+DG: Right, we only have witx.
+
+AB: Also generates md, which might count.
+
+SC: Implementation and test writing starts in stage 2. Probably don’t need a
+full vote. Does anyone have any comments or objections?
+
+PH: I followed it and I’m happy with how it looks.
+
+SC: Unanimous consent achieved to move to stage 2.
+
+AB: I will submit a PR updating which table it is in.
+
+#### Character encodings
```

Comment:
@sunfishcode can you upload your slides and add a link to them when you get a chance?